mysqlcluster/syncer/statefulset.go

```diff
@@ -590,9 +590,14 @@ func (s *StatefulSetSyncer) backupIsRunning(ctx context.Context) (bool, error) {
 
 // Updates to statefulset spec for fields other than 'replicas', 'template', and 'updateStrategy' are forbidden.
 func (s *StatefulSetSyncer) sfsUpdated(existing *appsv1.StatefulSet) bool {
+	var resizeVolume = false
+	if len(existing.Spec.VolumeClaimTemplates) > 0 || len(s.sfs.Spec.VolumeClaimTemplates) > 0 {
```

Comment:
MayBe it is correct now, should add TODO comment show that it will impove in future.